tests/test_metrics/test_metrics.py

```diff
@@ -210,6 +212,59 @@ def test_metrics_values(metric_class, metric_fn, train_test_dfs):
         assert value == true_metric_value
 
 
+def _create_metric_class(metric_fn, metric_fn_signature, greater_is_better):
```

Comment:
Too complicated test imho

Reply:
I know, but it is the easiest way I came up with to test that old signagure and new signature give the same results. May be we could narrow this somehow.